secrets/localsecrets/localsecrets_test.go

```diff
@@ -25,7 +25,7 @@ import (
 type harness struct{}
 
 func (h *harness) MakeDriver(ctx context.Context) (driver.Keeper, error) {
-	return NewKeeper(ByteKey("very secret secret")), nil
+	return &keeper{secretKey: ByteKey("very secret secret")}, nil
```

Comment:
Let's keep using NewKeeper so that it got covered by tests.

Reply:
We can't: `NewKeeper` returns a `*secrets.Keeper`, and `MakeDriver` needs to return a `driver.Keeper`. (It worked before because the concrete and driver interfaces happened to be the same.)